include/eve/module/polynomial/detail/horner_impl.hpp

```diff
@@ -24,6 +25,7 @@ namespace eve::detail
 template<decorator D, value T0, value... Cs>
 EVE_FORCEINLINE constexpr auto
 horner_impl(D const& d, T0 const& xx, Cs... cs) noexcept
+-> common_value_t<T0, Cs...>
 {
   using r_t          = common_compatible_t<T0, Cs...>;
```

Comment:
`using r_t          = common_compatible_t<T0, Cs...>;` 
should be
`using r_t          = common_value_t<T0, Cs...>;`

and then just use `auto`

Reply:
done